certora/specs/StakingRouterInvariants.spec

```diff
@@ -11,6 +11,7 @@ invariant StakingModuleIdLELast(uint256 moduleId)
 
 invariant StakingModuleIndexIsIdMinus1(uint256 moduleId)
     getStakingModuleIndexById(moduleId)+1 == getStakingModuleIdById(moduleId)
+    //ND : what about moduleIds that are not in the system? Is there some array reference here? looks like a limited-scope rule 
```

Comment:
It's valid for every staking module that is registered in the system. Otherwise, the index of the moduleId is just zero.